src/components/common/ServicesTable/ServiceTable.js

```diff
@@ -311,40 +315,42 @@ const ServicesTable = ({
         );
       },
     },
-    {
-      title: 'Notes',
-      dataIndex: 'notes',
-      key: 'notes',
-      editable: true,
-      render: (_, record) => {
-        const editable = isEditing(record);
-        return editable ? (
-          <Form.Item
-            name="notes"
-            style={{ margin: 0 }}
-            initialValue={record.service_entry_data.default.Notes}
-            rules={[
-              {
-                required: false,
-                message: 'Please select a notes',
-              },
-            ]}
-          >
-            <TextArea
-              value={record.service_entry_data.default.Notes}
-              rows={10}
-            />
-          </Form.Item>
-        ) : (
-          <>
-            {record.service_entry_data.default.Notes
-              ? record.service_entry_data.default.Notes
-              : 'N/A'}
-          </>
-        );
-      },
-    },
+    // Based on consensus, we think it makes for easier visibility / sorting to not make notes accessible on this table, but instead on a modal that allows you to edit all the
+    // fields on the service table for a specific record, which will be tackled in a seperate user story / trello card.
```

Comment:
Should we leave this code commented out?
Or should we just save it locally removed from the main branch?

Reply:
Honestly I think we should leave it commented out until stakeholder approves, agree?